src/main/java/net/discordjug/javabot/systems/moderation/report/ReportManager.java

```diff
@@ -147,16 +200,26 @@ protected WebhookMessageCreateAction<Message> handleUserReport(InteractionHook h
 				return;
 			}
 			reportChannel.sendMessageEmbeds(embed.build())
-					.queue(m -> this.createReportThread(m, target.getIdLong(), config.getModerationConfig()));
-			embed.setDescription("Successfully reported " + "`" + UserUtils.getUserTag(target) + "`!\nYour report has been send to our Moderators");
-			hook.sendMessageEmbeds(embed.build()).queue();
+					.queue(m -> this.createReportThread(m, target.getIdLong(), config.getModerationConfig(),
+							reportThread -> {
+								sendReportResponse(hook, target, embed, reportThread);
+							}));
 		}, failure -> {
 			Responses.error(hook, "The user to report seems not to exist any more.").queue();
 			log.warn("Cannot retrieve user {} when reporting them", targetId, failure);
 		});
 		return null;
 	}
 
+	private void sendReportResponse(InteractionHook hook, User targetUser, EmbedBuilder reportEmbed, ThreadChannel reportThread) {
+		reportEmbed.setDescription("Successfully reported " + "`" + UserUtils.getUserTag(targetUser) + "`!\nYour report has been send to our Moderators.\nIn case you want to supply additional details, please use the \"Create thread\" button below.");
+		hook.sendMessageEmbeds(reportEmbed.build())
+			.addActionRow(Button.secondary(
+					ComponentIdBuilder.build(REPORT_INTERACTION_NAME, "create-thread",reportThread.getId()),
+					"Create thread"))
```

Comment:
I'd rename this to something like `Create thread for further discussion` or similar, to make the purpose of this more clear.

Reply:
I don't like using the word `discussion` as that weakly implies that we would generally respond in the threads after handling the report. I rather see these threads as a way for the person reporting to include attachments or other relevant things (e.g. explain a complicated situation better).
I now changed the button to `Create thread for providing further details`.
![image](https://github.com/Java-Discord/JavaBot/assets/34687786/a0543785-9f3b-4afc-ba54-d5dcf98d53a5)